products/models.py

```diff
@@ -0,0 +1,137 @@
+from django.db import models
+
+from core import models as core_models
+
+
+class Menu(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "menus"
+
+    def __str__(self):
+        return self.name
+
+
+class Skin_Category(models.Model):
+    menu = models.ForeignKey("Menu", on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "skin_category"
+
+    def __str__(self):
+        return self.name
+
+
+class Genre_Category(models.Model):
+    menu = models.ForeignKey("Menu", on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "genre_category"
+
+    def __str__(self):
+        return self.name
+
+
+class Line_Category(models.Model):
+    menu = models.ForeignKey("Menu", on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "line_category"
+
+    def __str__(self):
+        return self.name
+
+
+class Online_Category(models.Model):
+    menu = models.ForeignKey("Menu", on_delete=models.CASCADE)
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "online_category"
+
+    def __str__(self):
+        return self.name
+
+
+class Product_Plag(models.Model):
+    flag_sale = models.CharField(max_length=10)
+    flag_gift = models.CharField(max_length=10)
+    flag_new = models.CharField(max_length=10)
+    flag_best = models.CharField(max_length=10)
+
+    class Meta:
+        db_table = "product_flag"
+
+    def __str__(self):
+        return self.name
+
+
+class Image(models.Model):
+    image_url = models.TextField(max_length=1000)
+
+    class Meta:
+        db_table = "image"
+
+
+class Product_Detail_Image(models.Model):
```

Comment:
클래스 이름에는 언더바를 사용하지 않습니다. 대문자로 띄어쓰기가 구분되니 언더바는 삭제해주세요
```suggestion
class ProductDetailImage(models.Model):
```